galaxy_ng/app/management/commands/update-synclist-distros.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from pulp_ansible.app.models import AnsibleDistribution, AnsibleRepository
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """This command updates all AnsibleDistribution in the format of #####-synclists
+    to point to the published repo.
+
+    Example:
+    django-admin update-synclist-distros
```

Comment:
I tested the management command on an ephemeral env with many distros. Pointing all to the published repo looked good. I removed the option to point all to a synclist repo, since some (most) distros already point to published and that command would undo that.

Reply:
The "undo" was initially to support a potential rollback scenario where the synclists might need to be undone. It was )or intended to ) checking if the synclist should point to a synclist repo aside from published. ie, it checked if 'excludes' was populated at all, and if so then it would be pointed to the synclist repo and otherwise left alone. In theory it shouldn't ever point all to a synclist.

That is all assuming we need/want a undo/rollback counterpart in the command. If we don't, then no need to include it.

Reply:
yeah i don't think the rollback is needed here afterall